src/components/selectable/selectable.tsx

```diff
@@ -382,6 +382,11 @@ export class EuiSelectable<T = {}> extends Component<
     }
   };
 
+  scrollToItem = () => {
+    const optionsList = this.optionsListRef.current;
+    if (optionsList) return optionsList.setListRef(optionsList.listRef);
+  };
```

Comment:
Sorry for the confusion, I incorrectly mentioned using `setListRef` instead of `listRef` in the issue. This should be,

```suggestion
  scrollToItem = (index: number, align?: Align) => {
    this.optionsListRef.current?.listRef?.scrollToItem(index, align);
  };
```

(needs to import `Align` from `react-window` as well)

Reply:
Thanks for the input. I have pushed a new commit with suggested changes. Please review it.